nemo/collections/nlp/modules/common/transformer/transformer.py

```diff
@@ -183,6 +184,10 @@ def __init__(
         self._vocab_size = vocab_size
         self._hidden_size = hidden_size
         self._max_sequence_length = max_sequence_length
+        self.num_states = num_layers + 1
```

Comment:
Is this something that needs to go in the TransformerConfig class or something that can be computed on the fly during export?